src/main/java/net/dv8tion/jda/internal/requests/restaction/ChannelActionImpl.java

```diff
@@ -60,6 +60,7 @@
                                                                               ChannelType.GUILD_PUBLIC_THREAD, ChannelType.GUILD_NEWS_THREAD, ChannelType.GUILD_PRIVATE_THREAD);
     private static final EnumSet<ChannelType> NSFW_SUPPORTED = EnumSet.of(ChannelType.TEXT, ChannelType.VOICE, ChannelType.FORUM, ChannelType.NEWS);
     private static final EnumSet<ChannelType> TOPIC_SUPPORTED = EnumSet.of(ChannelType.TEXT, ChannelType.FORUM, ChannelType.NEWS);
+    private static final EnumSet<ChannelType> DEFAULT_LAYOUT_SUPPORTED = EnumSet.of(ChannelType.FORUM);
```

Comment:
This isn't needed since its always going to be limited to forums only, simply check for just this type in the setter. This also has to be added to the `ForumChannelManager`.

Reply:
For the `ForumChannelManager`, where this has to be added ?

Reply:
Add the setter declaration inside the interface, the implementation should be in [`ChannelManagerImpl`](https://github.com/DV8FromTheWorld/JDA/blob/master/src/main/java/net/dv8tion/jda/internal/managers/channel/ChannelManagerImpl.java).

Reply:
It's done (normally)